src/status_im/events.cljs

```diff
@@ -149,6 +149,11 @@
  (fn [cofx _]
    (multiaccounts/confirm-wallet-set-up cofx)))
 
+(handlers/register-handler-fx
+ :multiaccounts.ui/hide-home-tooltip
+ (fn [cofx _]
+   (multiaccounts/confirm-home-tooltip cofx)))
```

Comment:
why not use `multiaccounts.update/multiaccount-update` directly here?

Reply:
yenda commented that it should follow 'wallet-set-up-passed?'. : https://github.com/status-im/status-react/pull/9170#discussion_r343575326